nano/node/transport/tcp.hpp

```diff
@@ -25,6 +25,28 @@ class tcp_message_item final
 	nano::account node_id;
 	std::shared_ptr<nano::transport::socket> socket;
 };
+
+class tcp_message_manager final
```

Comment:
Could probably be in its own file.

Reply:
I don't want to spend to much time on reworking this manager, since I plan to move this functionality to a message_processor class anyway.